src/styles/blocks/header-nav.scss

```diff
@@ -0,0 +1,58 @@
+.nav {
+  &__list {
+    display: flex;
+    text-align: center;
+    list-style: none;
+    text-decoration: none;
+    margin: 0;
+    padding: 0;
+
+    &__item {
+      text-align: center;
+      display: flex;
+    }
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    position: relative;
+    height: 60px;
+    margin-left: 20px;
```

Comment:
Using `margin-left: 20px;` on the `&__link` might cause alignment issues if the links are not the first child. Consider using a more flexible layout solution, like `justify-content` or `gap` on the parent flex container.